parts/linux/cloud-init/artifacts/labels.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Update Labels for Kubernetes nodes
+After=kubelet.service
+[Service]
+Restart=always
+RestartSec=60
```

Comment:
So we could not make this a service and just call kubectl label from ensureLabels. Then it would only happen on reimage as opposed to constantly. Constant has it's own allure. I think it only overwrites labels it's given it doesn't blank non colliding labels.